src/jit/codegenxarch.cpp

```diff
@@ -6949,7 +6949,7 @@ void CodeGen::genCkfinite(GenTree* treeNode)
         {
             inst_RV_RV(ins_Copy(targetType), targetReg, op1->gtRegNum, targetType);
         }
-        inst_RV_RV_IV(INS_shufps, EA_16BYTE, targetReg, targetReg, 0xb1);
+        inst_RV_RV_IV(INS_shufps, EA_16BYTE, targetReg, targetReg, (int8_t)0xb1);
```

Comment:
I believe I've caught the cases that are doing this for the SIMD instructions, but it looks like we frequently pass in unsigned values to `ival`. I wonder if there are other cases where such inputs are incorrectly interpreted as "does not fit in a byte".